xsuite/src/cli/newWalletCmd.ts

```diff
@@ -37,19 +44,45 @@ const action = async ({
       return;
     }
   } else {
-    keystore = Keystore.createFile_unsafe(walletPath, password);
+    if (pemPath === undefined && oldWalletPath === undefined) {
```

Comment:
```
let data;
if pemPath then set data with pem content
if oldWalletPath then set data with old wallet content
Keystore.createFile_unsafe(walletPath, password, data)
```

This works in all cases. Additionally an error should be raised in case both arguments are specified